src/packages/@ncigdc/utils/filters/prepared/significantConsequences.js

```diff
@@ -10,18 +10,14 @@ export const colorMap = {
     stop_gained: '#2E7D32',
   },
   cnv: {
-    'Deep Loss': '#64b5f6',
-    'Shallow Loss': '#64b5f6',
+    Loss: '#64b5f6',
```

Comment:
when you resolve the conflicts here, keep the commented out values I added in cnv-next